eng/common/sdl/packages.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Guardian.Cli" version="0.199.0"/>
+  <package id="Microsoft.Guardian.Cli" version="0.109.0"/>
```

Comment:
Is this a downgrade?

Reply:
This appears to have originated from arcade flow #8981. @missymessa Do you know if this change was expected?